src/System.Net.Http/src/netcore50/System/Net/HttpClientHandler.cs

```diff
@@ -167,99 +168,48 @@ public bool PreAuthenticate
 
         public bool UseDefaultCredentials
         {
-            get { return Credentials == null; }
+            get { return false; }
             set
             {
                 CheckDisposedOrStarted();
-                if (value)
-                {
-                    // System managed
-                    _rtFilter.ServerCredential = null;
-                }
-                else if (_rtFilter.ServerCredential == null)
-                {
-                    // The only way to disable default credentials is to provide credentials.
```

Comment:
Wasn't this comment true? If yes, it would allow apps to disable DefaultCredentials.